README.md

```diff
@@ -1,15 +1,88 @@
+- Knowledge of each cluster's network configuration.
+- Worker node IPs on all the clusters must be outside of the cluster/service CIDR ranges.
+
+An example of three clusters configured to use with Submariner would look like the following:
+
+| Cluster Name | Provider | Cluster CIDR | Service CIDR | DNS Suffix |
+|:-------------|:---------|:-------------|:-------------|:-----------|
+| broker       | AWS      | 10.42.0.0/16 | 10.43.0.0/16 | cluster.local |
+| west         | vSphere  | 10.0.0.0/16  | 10.1.0.0/16  | west.local |
+| east         | OnPrem   | 10.98.0.0/16 | 10.99.0.0/16 | east.local |
+
+## Installation
+
+### Download
+Download the latest binary for your operating system and architecture, then install it on your path. The binary has no external dependencies.
+
+To install for Linux amd64 in `~/.local/bin`:
+
+    mkdir -p ~/.local/bin
+    wget https://github.com/submariner-io/submariner-operator/releases/download/v0.1.0/subctl-v0.1.0-linux-amd64
+    install subctl-v0.1.0 -linux-amd64 ~/.local/bin/subctl
+    rm subctl-v0.1.0-linux-amd64
+If `~/.local/bin` is on your PATH, you will then be able to run subctl.
+
+## Deployment
+
+### Submariner
+
+To deploy submariner in a two Kubernetes cluster setup follow the below steps
+
+* Deploy Broker with Dataplane - To deploy the Submariner broker along with the data cluster run
+```
+subctl deploy-broker --kubeconfig <PATH-TO-KUBECONFIG-BROKER> --dataplane
+```
+
+
+* Join Clusters with Broker - To join all the other clusters with the broker cluster, run using the broker-info.subm generated in the folder from which the previous step was run.
+
+```
+subctl join --kubeconfig <PATH-TO-KUBECONFIG-DATA-CLUSTER> broker-info.subm
+```
+
+### Submariner with Service Discovery
+
+ **Project status**: The Lighthouse project is meant only to be used as a development preview. Installing the operator on an Openshift cluster may disable some of the operator features.
+
+The [Lighthouse](https://github.com/submariner-io/lighthouse) project helps in cross-cluster service discovery. It has the below additional dependencies
+
+- kubefedctl installed ([0.1.0-rc3](https://github.com/kubernetes-sigs/kubefed/releases/tag/v0.1.0-rc3)).
+- kubectl installed.
+
+To deploy Submariner with Lighthouse follow the below steps.
+
+* Deploy Broker with Dataplane - To deploy the Submariner broker along with the data cluster, run using the broker-info.subm generated in the folder from which the previous step was run.
+```
+subctl deploy-broker --kubeconfig <PATH-TO-KUBECONFIG-BROKER> --dataplane --service-discovery --broker-cluster-context <BROKER-CONTEXT-NAME>
+```
+
+* Join Clusters with Broker - To join all the other clusters with the broker cluster run using the broker-info.subm generated in the folder from which the previous step was run.
+
+```
+subctl join --kubeconfig <PATH-TO-KUBECONFIG-DATA-CLUSTER> broker-info.subm  --broker-cluster-context <BROKER-CONTEXT-NAME>
```

Comment:
Does it come instead of `--clusterid` option ?
Can it work in conjunction with `--clusterid` ?

Reply:
yes it do , I updated the doc.